js/utils/templateHelpers.js

```diff
@@ -50,16 +50,19 @@ export function parseEmojis(text, className = '', attrs = {}) {
 
 /**
  * If the average is a number, show the last 2 digits and trim any trailing zeroes.
- * Don't show the count if the count is invalid, x (0) would be inaccurate and confusing.
- * If the average is invalid or 0, don't show anything.
  * @param {number} average - the average rating
  * @param {number} count - the number of ratings
+ * @param {boolean) skipCount = a count wasn't sent, don't show it or test it for validity
  */
-export function formatRating(average, count) {
+export function formatRating(average, count, skipCount) {
```

Comment:
Normally the best practice with options is to us an options object, otherwise if more options come, it could make for a long signature to pass them in individually. Also, if you happen to only want to pass in the last option, you have to throw in a bunch of nulls.

No big deal to change this. More as an FYI for going forward.

Reply:
Agreed. I mostly didn't want to go back and change every place it's called, since every other place passes in the first two parameters.